src/core/managers/internal/ToastManager/helpers/tests/baseTransactionMock.ts

```diff
@@ -0,0 +1,16 @@
+import { ITransactionListItem } from 'lib/sdkDappCoreUi';
+import { TransactionServerStatusesEnum } from 'types';
+
+export const baseTransactionMock: ITransactionListItem = {
+  status: TransactionServerStatusesEnum.success,
+  asset: null,
+  action: { name: 'Transfer' },
+  link: 'https://explorer.example.com/tx/123',
+  hash: '123',
+  details: {
```

Comment:
why use details here instead of simply adding the members in the same object?
and sender instead of initiator
why not use the same mechanism like today for directions ?